### ISSUE TITLE
Swapping the position of the Fn key.

### ISSUE THREAD

**Issue by Dadoum** [aigilea/hp_spectre_x360_14_eu0xxx#21]
Sorry, this is not an issue but since there is no discussion tab I will ask it here;

Is it possible to modify the ACPI tables to swap the Right Ctrl/<> and Fn keys on the keyboard? Or any way to control it in software?

HP for some reason decided to use ANSI keyboard layout in Europe, but in consequence they moved the extra key on the opposite end of the keyboard (overwriting the Right Ctrl key). At the same time, the keys from the extra column they used to have in previous Spectre laptops (PageUp, PageDown, End and Home) collapsed into function keys on the right of the keyboard. Thus, I would like to swap those two keys. But since the HP BIOS is not very featureful, I am seeking alternate solutions, but I can't find much documentation on the laptop outside of here.

Anyway, thank you for your project, it made the laptop awesome for use with Linux.

**Comment by alaincou** [comment]
I think that already asking HP to modify the bios is a good thing, and if there are many requests, they will do it.
I would make a request for that

**Comment by Dadoum** [comment]
Do you know where feature request can be made? I posted something in the Hardware issue category but I don't think they will be able to escalate the problem to BIOS developers.

**Comment by n3thshan** [comment]
yes please let us know the link. Although i personally dont have this problem, I am willing to help!

**Comment by alaincou** [comment]
Good evening, I am in France and I use google translate, sorry...
Already, I am on the bios F.12 that I just installed, with the same errors.
HP states that "This package provides an update to the system BIOS. The update installs on supported computers running a supported Microsoft Windows operating system. Restart the computer to complete the installation."
Which indicates that HP does not support Linux or other systems.
However, not supporting does not imply not accepting Linux.
In short, these are decisions of HP decision-makers and for that, you just have to contact HP as a buyer on the support to ask to be compatible with Linux.
Personally, I use Debian in unstable mode, I also put Mint, and tested Fedora and Ubuntu, but I would also like for example to put Android to use as a tablet with the pen.
It's up to us to say what we want...
